### DISCUSSION TITLE
Regex Manager contains disallowed fields: postUpgradeTasks

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#22093] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitLab EE

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

post Upgrade Task is issued for a specific regex manager
And, schema error is identified by vscode while editing the file.

### What you tried so far.

This is my config.js file:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":dependencyDashboard"
  ],
  "automerge": false,
  "regexManagers": [
    {
      "fileMatch": [
        "^.sha.txt$"
      ],
      "currentValueTemplate": "c9s",
      "matchStrings": [
        "^(?<currentDigest>.*)(\n|$)"
      ],
      "depNameTemplate": "redhat/centos-stream/rpms/man-pages",
      "registryUrlTemplate": "https://gitlab.com",
      "datasourceTemplate": "gitlab-tags",
      "postUpgradeTasks": {
        "commands": [
          "./.gitlab/process.py"
        ],
        "fileFilters": [
          "*"
        ],
        "executionMode": "update"
      }
    }
  ]
}
```

The miss configuration was not identified by vscode even though the schema is mentioned. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
postUpgradeTasks should be in packageRules, not within regexManagers

**Reply by lmilbaum** [reply]
Got it. The documentation wasn't that clear about it. Thanks.

**Comment by lmilbaum** [comment]
BTW, should I see in the log file that the post Upgrade Task was invoked?

**Reply by rarkins** [reply]
Yes, you can see some logger commands in https://github.com/renovatebot/renovate/blob/ebf064e75bf98b01b59c37d62212fb2388fcd8aa/lib/workers/repository/update/branch/execute-post-upgrade-commands.ts#L26

**Reply by lmilbaum** [reply]
Thanks @rarkins. Sorry for the noise. I should have chased it myself.
